pages/_app.mdx

```diff
@@ -3,25 +3,33 @@ import { NodeDataProvider } from '../providers/nodeDataProvider';
 import { LocaleProvider } from '../providers/localeProvider';
 import { SiteProvider } from '../providers/siteProvider';
 import sourceSansPro from '../util/sourceSansPro';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
 
 import '../styles/old/index.scss';
 
+const theme = createTheme({
+  typography: {
+    fontFamily: [
+      'Source Sans Pro',
```

Comment:
You should probably use the sourceSansPro variable that is imported as it gives you the correct font family name 👀

Reply:
Like replace the first line with sourceSansPro.style.fontFamily